_contentTemplates/gantt/state.md

```diff
@@ -0,0 +1,608 @@
+#set-sort-from-code
+@*Set sorting from code through the Gantt state*@
+
+@using Telerik.DataSource
+
+<TelerikButton ThemeColor="primary" OnClick="@SetGanttSort">Set sort from code</TelerikButton>
+
+<TelerikGantt Data="@Data"
+              @ref="@GanttRef"
```

Comment:
`@ref` should come first, according to the style guide, especially when the state is on focus here.
```suggestion
<TelerikGantt @ref="@GanttRef"
              Data="@Data"
```